packages/blob/src/helpers.ts

```diff
@@ -1,14 +1,17 @@
 // common util interface for blob raw commands, not meant to be used directly
 // this is why it's not exported from index/client
 
-import { type Response } from 'undici';
-
 export interface BlobCommandOptions {
   /**
    * Define your blob API token.
    * @defaultvalue process.env.BLOB_READ_WRITE_TOKEN
    */
   token?: string;
+  /**
+   * The amount of times to retry the API requests if they fail.
+   * @defaultvalue 10
+   */
+  retries?: number;
```

Comment:
I wonder if this is worth exposing to customers. For the first version, I would not expose it just yet and see when someone wants it. As this would give us context.

Reply:
i also exposed it so i can disable it for some negative tests that should directly fail

Reply:
I see you may have exposed it for testing purposes? If that's the case let's make it so the unit test replies 10 times with a throw.

Why I don't want to expose it yet: it's easier to add than remove in APIs, so until we're sure there's a need to change the number of retries, let's not expose it.

Reply:
the problem wasn't that it threw 10 times. The test just always timed out. I thought about other ways to control the retry from outside like env vars, but i think thats more dangerous than having this explicit option

Reply:
yes. I guess we could make the test long running but than the CI need unnecessary long

Reply:
Ohh right because the retries are time based, can we advance time with Jest? Since this is a unit test, it could work fine.

Otherwise:
> but i think thats more dangerous than having this explicit option

We do this with `VERCEL_BLOB_API_URL` and it's working fine, perhaps we can still go this route?

Reply:
it doesn't work with the fake timers, so I added `VERCEL_BLOB_RETRIES`